qcfractal/interface/collections/optimization_dataset.py

```diff
@@ -69,15 +77,25 @@ def add_specification(self,
             A full quantum chemistry specification for Optimization
         description : str, optional
             A short text description of the specification
+        protocols : Optional[Dict[str, Any]], optional
+            Protocols for this specification.
         overwrite : bool, optional
             Overwrite existing specification names
 
+        Returns
+        -------
+        None
+            Description
+
```

Comment:
This is an odd bit of documentation to include. At least `Description` should be deleted.

Reply:
Whoops, auto doc got me.